lib/web3auth_flutter.dart

```diff
@@ -138,7 +111,11 @@ class Web3AuthFlutter {
     }
   }
 
-  static Future<void> launchWalletServices(LoginParams loginParams, ChainConfig chainConfig, {String path = "wallet"}) async {
+  static Future<void> launchWalletServices(
+    LoginParams loginParams,
```

Comment:
i released latest android sdk. this loginParams can be removed now

Reply:
Updated